scene/gui/graph_edit.cpp

```diff
@@ -30,8 +30,12 @@
 
 #include "graph_edit.h"
 
+#include "modules/visual_script/visual_script_editor.h"
 #include "os/input.h"
 #include "os/keyboard.h"
+#ifdef TOOLS_ENABLED
+#include "property_selector.h"
```

Comment:
No dependencies like this should happen, I suggest you find a way to extend GraphEdit to be useful for what you need.

Reply:
Resolved this issue by turning the dependency to a signal call.